src/index.ts

```diff
@@ -1 +1,8 @@
-export { parseTemplates } from './parse-templates';
+export { parseTemplates, DEFAULT_PARSE_OPTIONS } from './parse-templates';
+export {
+  isStrictMode,
+  isSupportedScriptFileExtension,
+  TEMPLATE_TAG_NAME,
+  TEMPLATE_LITERAL_MODULE_SPECIFIER,
+  TEMPLATE_LITERAL_IDENTIFIER,
```

Comment:
I think we can probably drop these three (`TEMPLATE_TAG_NAME`, `TEMPLATE_LITERAL_MODULE_SPECIFIER`, and `TEMPLATE_LITERAL_IDENTIFIER`). I don't think they should probably be needed by folks if they can just use `DEFAULT_PARSE_OPTIONS`?

Reply:
updated good call.